app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/channels/DefaultChannelAdapter.kt

```diff
@@ -69,6 +69,7 @@ class DefaultChannelAdapter(
 
                 val removeStr = context.resources.getString(R.string.homescreen_tile_remove)
 
+                // FIXME: We need to localize this, but are waiting until we hear back from UX
```

Comment:
Nit: "waiting until we hear back from UX" -> "waiting until we have the final string from UX"